examples/tree_2d_dgsem/elixir_euler_astro_jet_amr.jl

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
You don't need to append `_amr` for this example. We usually do this if there is another example without AMR (to distinguish two setups) or if AMR is the main feature of an elixir. Since this is an application, we don't need the `_amr` suffix here, I think. (needs to be reflected in the tests)

Reply:
Ok, I see.

Reply:
I wanted to add _amr in case I am looking for shocks with amr examples...it is easier to find then :-)

Reply:
Okay :+1: